src/app/core/config/config.service.ts

```diff
@@ -395,3 +396,20 @@ const addDefaultNoteDetailsConfig: ConfigMigration = (key, configPart) => {
 
   return configPart;
 };
+
+const migrateGroupByConfig: ConfigMigration = (key, configPart) => {
+  // Check if we are working with the EntityCountDashboard component and within the 'config' object
+  if (
+    key === "config" &&
```

Comment:
This is still not fully fool-proof. As I said: You should check if there is a property "component" with value "EntityCountDashboard". Only then you can be certain that the "config" property next to that really is the type of config you are looking to migrate.